nservicebus/rabbitmq/configuration-api.md

```diff
@@ -187,9 +202,17 @@ In Versions 4 and above xml configuration options for controlling lost connectio
 snippet:rabbitmq-custom-breaker-settings-code
 
 
-### Changing routing topology
+### Routing topology
```

Comment:
I am not sure if this belongs here or not, but do we need to have a section about migrating between topologies, how to go about it, do they need to do it all endpoint at the same time, do they need to purge all the queues, ...?